.travis.yml

```diff
@@ -25,7 +25,8 @@ env:
     - MOODLE_BRANCH=master
 
 before_install:
-  - phpenv config-rm xdebug.ini
+  # We want to verify that xdebug works of for coverage. Once we only support Moodle 3.10 and up, can switch our tests to pcov too.
```

Comment:
Hi @stronk7,
you know I'm not native English; IMHO `xdebug works of for coverage` should be `xdebug works for coverage`.

HTH,
Matteo

Reply:
BTW @stronk7,
LGTM!

[Nice to see the ref](https://github.com/moodlehq/moodle-plugin-ci/pull/166/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R22) to `XDEBUG_MODE` supported by _PHPUnit_ [since 7.0.12](https://github.com/sebastianbergmann/php-code-coverage/commit/52f55786aa2e52c26cd9e2db20aff2981e0f7399#diff-874f33563125619a7a5cb567ebe523c59258662d69858d78d24d947b275f9c6cR9).
Not sure if a [counterpart comment ](https://github.com/moodlehq/moodle-plugin-ci/pull/166/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R18) should be added here too.

HTH,
Matteo

Reply:
> IMHO `xdebug works of for coverage` should be `xdebug works for coverage`.

Heh, I think I wanted to say `works ok for coverage`, but will use your alternative.

> Not sure if a [counterpart comment ](https://github.com/moodlehq/moodle-plugin-ci/pull/166/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R18)should be added here too.

I think that's ok as is. Both places are "internal/CI" testing. And we only can use them (xdebug - or phpdbg) there until we bump to Moodle 3.10 (or PHPUnit 8.0, it's exactly the same) as minimum supported version. Then, happily, will move to pcov both places.